rollup.config.mjs

```diff
@@ -66,10 +79,14 @@ const configBundles = {
     // to have sizes of dependencies listed at the end of build log
     sizes(),
   ],
-  output: [{ file: pkg.module, format: 'es' }],
+  output: {
+    file: pkg.module,
+    format: 'es',
+    banner: computeBanner(),
```

Comment:
**thought:** this may have been done in the `add-license-header.mjs` script.
I choose to do it here as this doesn't change the sourceMap information when enabled. The bundle generation may also change soon as I would like to remove the whole comments as part of #2489.
I'd rather we wait for the other change before possibly changing the way we add headers here.